src/components/Accordion/Accordion.test.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+
+import { Accordion, AccordionProps } from './';
+
+const DEFAULT_PROPS: AccordionProps = {
+  items: [
+    {
+      title: 'Title',
+      text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Eget quam netus consectetur magnis. Lorem est ac duis risus semper. Nunc faucibus turpis ac sollicitudin suscipit imperdiet est, cursus.'
+    }
+  ]
+};
+
+beforeEach(() => {
+  jest.resetAllMocks();
+});
+
+describe('<Accordion />', () => {
+  test('Accordion must be in closed state by default', () => {
+    const { container } = render(<Accordion {...DEFAULT_PROPS} />);
+    const selectAccordionTriggerElement = container.getElementsByClassName('AccordionTrigger')[0];
+    expect(selectAccordionTriggerElement).toHaveAttribute('data-state', 'closed');
+  });
+
+  test('Accordion must be in open state when user clicks on accordion header', () => {
+    const { container } = render(<Accordion {...DEFAULT_PROPS} />);
+    const selectAccordionTriggerElement = container.getElementsByClassName('AccordionTrigger')[0];
+    fireEvent.click(selectAccordionTriggerElement);
+    expect(selectAccordionTriggerElement).toHaveAttribute('data-state', 'open');
+  });
+
+  test('Accordion must be in closed state when user clicks on accordion header two times', () => {
+    const { container } = render(<Accordion {...DEFAULT_PROPS} />);
+    const selectAccordionTriggerElement = container.getElementsByClassName('AccordionTrigger')[0];
+    fireEvent.click(selectAccordionTriggerElement);
+    expect(selectAccordionTriggerElement).toHaveAttribute('data-state', 'open');
+    fireEvent.click(selectAccordionTriggerElement);
+    expect(selectAccordionTriggerElement).toHaveAttribute('data-state', 'closed');
+  });
```

Comment:
Could combine these two - `should toggle "open" state on click`.

```
test 'should toggle "open" state on click'
    // select a closed toggle
    // click it
    // check that it's now open
    // click it
    // check that it's now closed
```

Reply:
Updated